### PR TITLE
feature/SOF 7530

### PR DESCRIPTION
**Pull request by VsevolodX**
- feat: add high-k tutorial (claude)
- update: create slabs before (claude)
- update: adjustments
- update: cleanup
- update: add images
- update: add original image
- update: add interactive notebook

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -232,6 +232,7 @@ nav:
                 - Interface between Graphene and hBN:                          tutorials/materials/specific/interface-2d-2d-graphene-boron-nitride.md
                 - Interface between Copper and SiO2 (Cristobalite):                  tutorials/materials/specific/interface-3d-3d-copper-silicon-dioxide.md
                 - Interface between Graphene and SiO2 (alpha-quartz):                tutorials/materials/specific/interface-2d-3d-graphene-silicon-dioxide.md
+                - High-k Metal Gate Stack:                                      tutorials/materials/specific/heterostructure-high-k-metal-gate-stack.md
```
Comment (timurbazhirov) [inline mkdocs.yml:235]: We should list the materials used: Si, SiO2, HfO2, TiN. Same for the name.

File: lang/en/docs/tutorials/materials/specific/heterostructure-high-k-metal-gate-stack.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
+# Creating High-k Metal Gate Stack: Si/SiO2/HfO2/TiN
+
+## Introduction
+
+This tutorial demonstrates how to create a high-k metal gate stack heterostructure consisting of four materials: Si (substrate), SiO2 (gate oxide), HfO2 (high-k dielectric), and TiN (metal gate). The process involves:
+1. Creating individual slabs for HfO2 and TiN
```
Comment (timurbazhirov) [inline lang/en/docs/tutorials/materials/specific/heterostructure-high-k-metal-gate-stack.md:6]: Needs a newline to show up as list

File: lang/en/docs/tutorials/materials/specific/heterostructure-high-k-metal-gate-stack.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+3. Adding the pre-created slabs sequentially using simple interface builder
+
+We use the [Materials Designer](../../../materials-designer/overview.md) to create the high-k metal gate stack as shown in the figure below.
+
+![High-k Metal Gate Stack](/images/tutorials/materials/heterostructures/heterostructure-high-k-metal-gate-stack/original-figure.webp "High-k Metal Gate Stack")
+
+## 1. Set Up Materials
+
+First, navigate to Materials Designer and import from [Standata](../../../materials-designer/header-menu/input-output/standata-import.md) the following materials:
+- Silicon (Si)
```
Comment (timurbazhirov) [inline lang/en/docs/tutorials/materials/specific/heterostructure-high-k-metal-gate-stack.md:17]: Same as above

**Review by timurbazhirov — APPROVED** [review]

File: mkdocs.yml (hunk 1 above)
Comment (VsevolodX) [reply mkdocs.yml:235]: `heterostructure-high-k-metal-gate-stack-silicon-silicon-dioxide-hafnium-dioxide-titanium-nitride.md` -- too long of a name?

**Pull request merged**